visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -402,6 +403,7 @@ void MotionPlanningFrame::remoteUpdateStartStateCallback(const std_msgs::EmptyCo
 {
   if (move_group_ && planning_display_)
   {
+    planning_display_->syncSceneUpdates();
     robot_state::RobotState state = *planning_display_->getQueryStartState();
```

Comment:
I would propose to move the sync down by one line (in both cases) to make it clear that the sync is because of `getPlanningSceneRO` and not because of `getQueryStartState`.
Also, this `getQueryStartState()` call seems to be totally superfluous to me. Maybe we could remove them too?